Grbl_Esp32/src/Report.h

```diff
@@ -22,8 +24,8 @@
 
 // Define status reporting boolean enable bit flags in status_report_mask
 enum RtStatus {
-    Position = bit(0),
-    Buffer   = bit(1),
+    Position = 0,
```

Comment:
This needs to be changed back to bit(0) and bit(1) "?" reports are not working correctly or 1 and 2

Reply:
I agree.  The numerical value of the status mask is part of the external interface that senders depend on, so we are not free to reassign the enumerated values.

Reply:
Must have been late when i changed this, the goal was not to change the api, just to make it compile again (circular dependencies).